packages/eslint-plugin/src/configs/strict.ts

```diff
@@ -0,0 +1,48 @@
+// THIS CODE WAS AUTOMATICALLY GENERATED
+// DO NOT EDIT THIS CODE BY HAND
+// YOU CAN REGENERATE IT USING yarn generate:configs
+
+export = {
+  extends: [
+    './configs/base',
+    './configs/eslint-recommended',
+    './configs/recommended',
+    './configs/recommended-requiring-type-checking',
+  ],
+  rules: {
+    '@typescript-eslint/array-type': 'warn',
+    '@typescript-eslint/ban-tslint-comment': 'warn',
+    '@typescript-eslint/class-literal-property-style': 'warn',
+    '@typescript-eslint/consistent-indexed-object-style': 'warn',
+    '@typescript-eslint/consistent-type-assertions': 'warn',
+    '@typescript-eslint/consistent-type-definitions': 'warn',
+    'dot-notation': 'off',
+    '@typescript-eslint/dot-notation': 'warn',
+    '@typescript-eslint/no-base-to-string': 'warn',
+    '@typescript-eslint/no-confusing-non-null-assertion': 'warn',
+    '@typescript-eslint/no-dynamic-delete': 'warn',
+    '@typescript-eslint/no-extraneous-class': 'warn',
+    '@typescript-eslint/no-invalid-void-type': 'warn',
+    '@typescript-eslint/no-meaningless-void-operator': 'warn',
+    '@typescript-eslint/no-non-null-asserted-nullish-coalescing': 'warn',
```

Comment:
wait - this isn't recommended?!?

Oh - I see - it was added after the last major!